pkgs/development/python-modules/mesa/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, isPy27
+, cookiecutter, networkx , pandas, tornado, tqdm
+, pytest }:
+
+buildPythonPackage rec {
+  pname = "mesa";
+  version = "0.8.6";
+
+  # According to their docs, this library is for Python 3+.
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    repo = "mesa";
```

Comment:
can you flip owner and repo? Usually, it's the other way round :-D

Reply:
Oh, sure. Don't know how that happened :laughing: